content/welcome.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Welcome
+---
+
+Welcome to the Meteor guide.
+
+This is a set of articles outlining opinions on best-practice application development in the [Meteor](https://meteor.com) framework. Our aim is that the set of articles detailed here covers patterns that are common to the development of all modern web and mobile applications. In this sense a lot of what's documented here is not necessarily Meteor specific and could be equally be applied to any application built with a focus on modern, reactive user interfaces.
+
+Nothing in the Meteor guide is *required* to build a Meteor application (i.e. you can certainly use the platform in ways that contradict the principles and patterns of the guide). However, the guide is an attempt to document what we consider are best practices for app development. The guide is also an effort to document community conventions, and so we expect that in the future the practices documented here will generally be followed by the majority of the Meteor community.
```

Comment:
Personally I like to avoid parens:

> Nothing in the Meteor guide is *required* to build a Meteor application. You can certainly use the platform in ways that contradict the principles and patterns of the guide.

Also, we say "framework" above and "platform" here. We should pick one and run with it; I'd say "platform" since that's what the website says?

----------

> However, the guide is an attempt to document what we consider are best practices for app development. The guide is also an effort to document community conventions, and so we expect that in the future the practices documented here will generally be followed by the majority of the Meteor community.

to:

"However, the guide is an attempt to document best practices and community conventions, so we hope that the majority of the Meteor community will benefit from adopting the practices documented here."

What do you think?